package.json

```diff
@@ -2,8 +2,8 @@
   "name": "circular-natal-horoscope-js",
   "version": "1.0.11",
   "description": "",
-  "main": "dist/index.js",
-  "module": "src/index.js",
+  "main": "dist/index.d.ts",
+  "module": "src/index.d.ts",
```

Comment:
I want to keep this compatible with non-typescript users - will this break for them?